src/creature_instances.c

```diff
@@ -752,7 +752,7 @@ long instf_first_person_do_imp_task(struct Thing *creatng, long *param)
         ahead_stl_y = creatng->mappos.y.stl.num;
         ahead_stl_x = creatng->mappos.x.stl.num + 1; 
     }
-    if ( (first_person_dig_claim_mode) || (!subtile_is_diggable_for_player(creatng->owner, ahead_stl_x, ahead_stl_y)) )
+    if ( (first_person_dig_claim_mode) || (!subtile_is_diggable_for_player(creatng->owner, ahead_stl_x, ahead_stl_y, false)) )
```

Comment:
setting `enemy_wall_undiggable` to true here prevents the digging sound when you are in possession mode (with an imp) trying to dig an enemy wall.

Not sure of any other impact...

Reply:
What's the alternative sound?

Reply:
No sound is played. 
I would think it works like this:
imp thinks it can dig it -> the digging sound is played
imp thinks it cannot dig it -> no sound is played

Reply:
You hear a sound when imps 'dig' rock.

Reply:
OK, well if you want the sound to stay - then this should be left as false.